compiler/tests/char/mod.rs

```diff
@@ -0,0 +1,93 @@
+// Copyright (C) 2019-2021 Aleo Systems Inc.
+// This file is part of the Leo library.
+
+// The Leo library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The Leo library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with the Leo library. If not, see <https://www.gnu.org/licenses/>.
+
+use crate::{assert_satisfied, get_output, parse_program, parse_program_with_input, EdwardsTestCompiler};
+
+pub fn output_char(program: EdwardsTestCompiler) {
+    let expected = include_bytes!("output/output_char.out");
+    let actual = get_output(program);
+
+    assert_eq!(expected, actual.bytes().as_slice());
+}
+
```

Comment:
Can we add some tests for character parsing failures here?

Reply:
Yup! I planned on that, and I just wanted to get some opinions on if I should continue writing compiler tests in this format or have a dependency on #920 first. I would rather have to convert as few tests as possible(or have @damirka convert less), but I'll do whatever people prefer.

Reply:
I also don't have a good way to run code coverage locally to see what kinds of tests I'm missing; anyone have good recommendations for this?